cfme/tests/services/test_different_dialogs_in_catalogs.py

```diff
@@ -477,7 +551,33 @@ def test_dialog_elements_should_display_default_value(element_type):
             1.
             2.
             3. Default values should be shown
-    Bugzilla:
-        1385898
     """
-    pass
+    sd, element_data = service_dialog
+    service_catalogs = ServiceCatalogs(
+        appliance, generic_catalog_item.catalog, generic_catalog_item.name
+    )
+    view = navigate_to(service_catalogs, "Order")
+    ele_name = element_data["element_information"]["ele_name"]
+    wt, value = WIDGETS[widget_name]
+
+    if widget_name in ["Text Box", "Text Area"]:
+        assert getattr(view.fields(ele_name), wt).read() == value
+    elif widget_name == "Check Box":
+        assert view.fields(ele_name).checkbox.read()
+    elif widget_name == "Tag Control":
```

Comment:
You are also testing Tag control here , see if you can merge the test_tagdialog_catalog_item test case . If you provide 'service_level' instead of 'Owner' in line 30 , I think it will become same test case

Reply:
No I can not merge these two test case. Because ```Tag Control``` test case is parametrized against providers catalog item and this new test case deals with generic catalog item only. 
Hence just using ```service_dialog``` fixture for both testcases